library/include/vulkan/vulkan_profiles.hpp

```diff
@@ -8109,6 +8109,18 @@ VPAPI_ATTR VkResult vpCreateInstance(const VpInstanceCreateInfo *pCreateInfo,
                                     pDesc->instanceExtensionCount,
                                     pDesc->pInstanceExtensions,
                                     extensions, merge, override);
+            {
+                bool foundPortEnum = false;
+                for (size_t i = 0; i < extensions.size(); ++i) {
+                    if (strcmp(extensions[i], VK_KHR_PORTABILITY_ENUMERATION_EXTENSION_NAME) == 0) {
+                        foundPortEnum = true;
+                        break;
+                    }
+                }
+                if (foundPortEnum) {
+                    createInfo.flags = VK_INSTANCE_CREATE_ENUMERATE_PORTABILITY_BIT_KHR;
```

Comment:
```suggestion
                    createInfo.flags |= VK_INSTANCE_CREATE_ENUMERATE_PORTABILITY_BIT_KHR;
```

Reply:
This is resolved in the following commit